src/aframe-streetmix-parsers.js

```diff
@@ -1726,7 +1722,6 @@ module.exports.processSegments = processSegments;
 function processBuildings(left, right, streetWidth, showGround, length) {
   const buildingElement = document.createElement('a-entity');
   const clonedObjectRadius = 0.45 * length;
-  buildingElement.classList.add('buildings-parent');
```

Comment:
Be careful, this class is checked in https://github.com/3DStreet/3dstreet/blob/2929a6d8b9aa5c1c2b3281fea2eb6659b26929e1/src/editor/components/components/AddLayerPanel/AddLayerPanel.component.jsx#L110-L111

Reply:
ok we'll change this later